core/domain/stats_services.py

```diff
@@ -169,6 +169,17 @@ def get_versions_for_exploration_stats(exploration_id):
         exploration_id)
 
 
+def get_total_unresolved_answers_for_exploration(exploration_id):
+    """Returns the sum total of unresolved answers (across all states) of an
+    exploration."""
+    exploration = exp_services.get_exploration_by_id(exploration_id)
+    state_names = exploration.states.keys()
+
+    return sum(stats_domain.StateRuleAnswerLog.get(
```

Comment:
The `get_multi` method will help, at it means you will only be doing 1 call to the database, rather than N calls (where N is the number of states). Each `get` call accesses the database once.